vms/platformvm/static_service.go

```diff
@@ -275,8 +275,8 @@ func (ss *StaticService) BuildGenesis(_ *http.Request, args *BuildGenesisArgs, r
 		switch {
 		case weight == 0:
 			return errValidatorAddsNoValue
-		case weight != uint64(args.ValidatorBondAmount):
-			return errWrongBondAmount
+			//case weight != uint64(args.ValidatorBondAmount):
```

Comment:
Not sure about this. In both colombus and camino genesis files, `unlockSchedule.amount` and  `validatorBondAmount` of the initial validators have the same value. But in genesis local these values are different  (20000000000000000 and 2000000000000) and this check fails.

Reply:
Then we should correct values in genesis local, so they'll be equal